frontend/src/lib/lemon-ui/LemonButton/More.tsx

```diff
@@ -1,14 +1,24 @@
 import { LemonButtonWithDropdown } from '.'
 import { IconEllipsis } from 'lib/lemon-ui/icons'
 import { PopoverProps } from '../Popover/Popover'
+import { LemonButtonWithDropdownProps } from './LemonButton'
 
 export interface MoreProps extends Partial<Pick<PopoverProps, 'overlay' | 'placement'>> {
     style?: React.CSSProperties
     className?: string
     'data-attr'?: string
+    tooltip?: string
```

Comment:
A tooltip doesn't really seem like a useful built-in feature for `More`. For the options within – sure – but not the three dots. Can we drop this?